tests/feature_elimination/test_feature_elimination.py

```diff
@@ -363,7 +403,7 @@ def test_shap_rfe_randomized_search_early_stopping(complex_data):
 
 
 @pytest.mark.skipif(os.environ.get("SKIP_LIGHTGBM") == "true", reason="LightGBM tests disabled")
```

Comment:
So far we have problems with lightbm tests on macos. They simply fail because we are not able to get the image needed for lightgbm them. This decorator is a temporary solutions to skip them for macos machines.

I see the similar issue happens for catboost model. That is why I would suggest to put the decorator in the 2 extra tests you added.

Reply:
Added, thanks for the tip